### PR TITLE
Fix e2e tests after separating admin api

### PR DESCRIPTION
**Pull request by tobiaszheller**
# What

This PR fixes E2E tests after separating admin from api.

# Details
It turned out that after all e2e tests were broken after moving `admin` to separate app. 
I forgot to run `docker compose build` before manually running them last time. (I will setup some CI in another PR).

It has following changes:
1. Use or shared volume in docker compose (previously admin and api endpoints were running in single container and icons were stored locally. On production it works because for storage is used S3. Here I have used shared volume in docker compose).
2. Update all e2e tests to use admin endpoints for "admin features"

### CODE REVIEW

File: internal/api/browser_extension/service/service.go [hunk 1]
```diff
@@ -148,12 +148,11 @@ func (m *BrowserExtensionModule) RegisterPublicRoutes(router *gin.Engine) {
...
 	publicRouter.POST("/browser_extensions/:extension_id/2fa_requests/:token_request_id/commands/close_2fa_request", m.RoutesHandler.Close2FaRequest)
 
-	if m.Config.IsTestingEnv() {
-		publicRouter.DELETE("/browser_extensions", m.RoutesHandler.RemoveAllBrowserExtensions)
-		publicRouter.DELETE("/browser_extensions/devices", m.RoutesHandler.RemoveAllBrowserExtensionsDevices)
-	}
-
 }
 func (m *BrowserExtensionModule) RegisterAdminRoutes(g *gin.RouterGroup) {
+	if m.Config.IsTestingEnv() {
```
Comment (tobiaszheller) [inline internal/api/browser_extension/service/service.go:153]: just to be consistent with other changes in #9 , all testing endpoints are under admin routes.

File: internal/common/storage/fs.go [hunk 2]
```diff
@@ -19,7 +19,7 @@ type TmpFileSystem struct {
 }
 
 func NewTmpFileSystem() FileSystemStorage {
-	tmpDir := "/tmp"
+	tmpDir := "/tmp/2fas"
```
Comment (tobiaszheller) [inline internal/common/storage/fs.go:22]: in docker compose I am mounting `/tmp/2fas` as shared volume.

File: docker-compose.yml [hunk 3]
```diff
@@ -69,5 +80,21 @@ services:
...
+    volumes:
+      - shared-volume:/tmp/2fas
+    command: chown -R 1000:1000 /tmp/2fas
+
+
 volumes:
   go-modules:
+  # shared-volume is used to share volume between api and admin. On producition AWS S3 is used, 
+  # but here for local dev shared volume is fine.
+  shared-volume:
```
Comment (krzysztofdrys) [inline docker-compose.yml:100]: new line

File: docker-compose.yml [hunk 4]
```diff
@@ -69,5 +80,21 @@ services:
...
+    user: root
+    group_add:
+      - '1000'
+    volumes:
+      - shared-volume:/tmp/2fas
+    command: chown -R 1000:1000 /tmp/2fas
+
+
 volumes:
   go-modules:
```
Comment (krzysztofdrys) [inline docker-compose.yml:97]: TBH it doesn't look as valid yaml. But if docker doesn't complain, I won't object it as well.

File: tests/http.go [hunk 5]
```diff
@@ -14,7 +14,10 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-const DebugHttpRequests = false
+const (
+	DebugHttpRequests = false
+	adminRawURL       = "http://localhost:8082/admin"
```
Comment (krzysztofdrys) [inline tests/http.go:19]: I don't like that we have `baseURL` and `adminRawURL` (raw vs non-raw). 

Is host the only difference between admin and non-admin? If so, could you add a common method? Something like:

```
func DoSuccessPost(t *testing.T, uri string, payload []byte, resp interface{}) {
  return doSuccessPost(t, baseURL, uri, payload, resp)
 }
 
 func DoAdminSuccessPost(t *testing.T, uri string, payload []byte, resp interface{}) {
  return doSuccessPost(t, adminURL, uri, payload, resp)
 }
 ```

**Review by krzysztofdrys — COMMENTED** [review]
Looks ok, though I left some small comment.

File: tests/http.go (hunk 5 above)
Comment (tobiaszheller) [reply tests/http.go:18]: in https://github.com/twofas/2fas-server/pull/11/commits/b721a8588ac90248b601c9d15f62a17e1de16941 i have reworked it, ptal

**Review by krzysztofdrys — APPROVED** [review]

**Pull request merged**